runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -267,8 +267,6 @@ static const char* getNameForMethodRelocation (int type)
             return "TR_VirtualRamMethodConst";
          case TR_ClassAddress:
             return "TR_ClassAddress";
-         case TR_StaticDefaultValueInstance:
```

Comment:
I see some uses of TR_StaticDefaultValueInstance in OMR as well that should be removed.